packages/liveblocks-react/src/room.tsx

```diff
@@ -124,6 +124,10 @@ const STABLE_EMPTY_LIST = Object.freeze([]);
 
 export const POLLING_INTERVAL = 5 * 60 * 1000; // 5 minutes
 
+export const ERROR_RETRY_INTERVAL = 5000; // 5 seconds
+
+export const MAX_ERROR_RETRY_COUNT = 5;
```

Comment:
`ERROR_RETRY_INTERVAL` represents the initial timeout for the error retrying, which is then exponentially incremented for `MAX_ERROR_RETRY_COUNT` number of times. These values are arbitrarily chosen, please feel free to suggest any other values that would make more sense.